__tests__/rpcProvider.test.ts

```diff
@@ -28,7 +25,8 @@ describeIfRpc('RPCProvider', () => {
     );
   });
 
-  test('getPendingTransactions', async () => {
+  xtest('getPendingTransactions', async () => {
```

Comment:
Could be wrapped in `describeIfNotDevnet`.

Reply:
a54c6ee25eef9494267400b3198c20585099a3e6